app-ios/README.md

```diff
@@ -135,9 +135,9 @@ graph LR
 
 ### Prerequisites
 
-- **Xcode**: 16.0 or later
+- **Xcode**: 16.4 or later
 - **Swift**: 6.1.2
-- **macOS**: 15.0 or later
+- **macOS**: 15.5 or later
```

Comment:
Based on the official sources and the links below, I believe that macOS version 15.3 or later would be required for this combination. Are there any specific sites or references you used for confirmation?

https://xcodereleases.com

<img width="1038" height="718" alt="Image" src="https://github.com/user-attachments/assets/f25cc494-32bf-473b-9d5c-146954c45dfd" />

https://developer.apple.com/jp/support/xcode/

<img width="900" height="603" alt="Image" src="https://github.com/user-attachments/assets/df1c6ea4-3053-4ce4-a97c-d73a158f7d99" />